### PR TITLE
Rephrasing description of agents

### PR DESCRIPTION
**Pull request by papagala**
This PR rephrases the description of an agent which was a bit ambiguous from:

- An agent is installable software that runs one job at a time

to 

- An agent is computing infrastructure that installs and runs software one job at a time

### CODE REVIEW

**Comment by PRMerger15** [comment]
@papagala : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/pipelines/agents/agents.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ monikerRange: '>= tfs-2015'
 To build your code or deploy your software using Azure Pipelines, you need at least one agent. As you add more code and people, you'll eventually need more.
 
 When your pipeline runs, the system begins one or more jobs.
-An agent is installable software that runs one job at a time.
+An agent is computing infrastructure that installs and runs software one job at a time.
```
Comment (steved0x) [inline docs/pipelines/agents/agents.md:24]:
```suggestion
An agent is computing infrastructure with installed agent software that runs one job at a time.
```

File: docs/pipelines/get-started/key-pipelines-concepts.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ Learn about the key concepts and components that are used in Azure Pipelines. Un
 
 ## Agent
 
-When your build or deployment runs, the system begins one or more jobs. An agent is installable software that runs one job at a time.
+When your build or deployment runs, the system begins one or more jobs. An agent is computing infrastructure that installs and runs software one job at a time.
```
Comment (steved0x) [inline docs/pipelines/get-started/key-pipelines-concepts.md:32]:
```suggestion
When your build or deployment runs, the system begins one or more jobs. An agent is computing infrastructure with installed agent software that runs one job at a time.
```

**Comment by steved0x** [comment]
@papagala Thank you for your contribution - I have suggested a small change to your suggestion. If you approve, please accept the changes and I will merge the PR. Thank you.

File: docs/pipelines/agents/agents.md (hunk 1 above)
Comment (papagala) [reply docs/pipelines/agents/agents.md:24]: I like it better as you suggest. It is even more explicit and clear

File: docs/pipelines/get-started/key-pipelines-concepts.md (hunk 2 above)
Comment (papagala) [reply docs/pipelines/get-started/key-pipelines-concepts.md:32]: Perfect

**Comment by PRMerger15** [comment]
@papagala : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger15** [comment]
@papagala : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by steved0x** [comment]
#sign-off @papagala Thank you for your contribution!

**Pull request merged**